src/test/java/fi/metatavu/metaform/server/test/functional/tests/MetaformTestsIT.kt

```diff
@@ -597,4 +597,33 @@ class MetaformTestsIT : AbstractTest() {
             builder.test2.metaforms.assertCount(0, visibility = null, memberRole = MetaformMemberRole.MANAGER)
         }
     }
+
+    @Test
+    fun testScheduledFields() {
+        TestBuilder().use { builder ->
```

Comment:
This could be simplified by using expression body syntax:

```suggestion
    fun testScheduledFields() = TestBuilder().use { builder ->
```